shells/zsh/Portfile

```diff
@@ -21,7 +21,7 @@ homepage                https://www.zsh.org
 depends_lib             port:gdbm \
                         port:libiconv \
                         port:ncurses \
-                        port:pcre
+                        port:pcre2
```

Comment:
Changing a library dependency requires a "revision" increase.

Reply:
And also usually some change in configure flags or other way of controlling which libraries are used.